benchmark/readline/readline-iterable.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+const common = require('../common.js');
+const readline = require('readline');
+const { Readable } = require('stream');
+
+const bench = common.createBenchmark(main, {
+  n: [1e1, 1e2, 1e3, 1e4, 1e5, 1e6],
+});
+
+const loremIpsum = `Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
```

Comment:
Let's wrap it to 80 chars, because why not
```suggestion
const loremIpsum = `Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
do eiusmod tempor incididunt ut labore et dolore magna aliqua.
```